File: src/features/formazione/corsi/sgiqas.ts
import { NUMERI } from "../numeri";
import type { Sezione } from "../tipi";

/** SGI QAS (qualità, ambiente, sicurezza). Metodo dal corso del committente. */
export const SGIQAS: Sezione[] = [
  {
    id: "anagrafica-qas",
    titolo: "Anagrafica, norme applicate e perimetro",
    minuti: 5,
    sommario: "Le norme dichiarate decidono che cosa conta, e le ore lavorate fanno gli indici.",
    blocchi: [
      {
        tipo: "avviso",
        tono: "nota",
        titolo: "Il perimetro decide quanti requisiti sono in gioco",
        testo: `Le norme applicate sono un elenco, non una scelta unica: si possono combinare, e da quella combinazione dipende quali dei **${NUMERI.requisitiQas} requisiti** entrano nella mappa. Chi è certificato su una norma sola non deve vedersi contare addosso i requisiti delle altre due.`,
      },
      {
        tipo: "tabella",
        intestazioni: ["Gruppo", "Che cosa muove"],
        righe: [
          [
            "Identificazione",
            "Unità locali e cantieri, addetti, e le **ore lavorate nel periodo**, che sono il denominatore degli indici infortunistici",
          ],
          ["Ruoli", "Direzione, responsabile del sistema, responsabile della prevenzione, medico competente, rappresentanti dei lavoratori"],
          [
            "Sistema",
            "Norme applicate e certificazioni, campo di applicazione, esclusioni motivate, e la determinazione sul cambiamento climatico come fattore rilevante",
          ],
        ],
      },
      {
        tipo: "avviso",
        tono: "errore",
        titolo: "Le esclusioni valgono solo dove la norma le ammette",
        testo:
          "Si può escludere un requisito di qualità non applicabile, per esempio la progettazione. Non esistono esclusioni per attività con impatti ambientali significativi, né per i luoghi di lavoro: chi le dichiara sta escludendo qualcosa che nessun organismo accetterà di escludere.",
      },
      {
        tipo: "avviso",
        tono: "attenzione",
        titolo: "Senza le ore lavorate gli indici restano vuoti",
        testo:
          "E restano vuoti nel cruscotto e nel riesame, che sono i due posti in cui vengono cercati. Vanno aggiornate a ogni periodo di rendicontazione, non una volta sola all'avvio.",
      },
    ],
  },

  {
    id: "indicatori-qas",
    titolo: "Gli indicatori",
    minuti: 8,
    sommario: `I ${NUMERI.indicatoriQas} indicatori del set di partenza, e le due cose che li fanno mentire.`,
    blocchi: [
      {
        tipo: "prosa",
        testo: `Ogni indicatore ha una definizione, una formula, una fonte del dato, una frequenza, un valore iniziale, un target con il proprio **verso di miglioramento** e una soglia di attenzione. Il set di partenza ne propone **${NUMERI.indicatoriQas}**, distribuiti fra qualità, ambiente, sicurezza e integrati, con formula e riferimenti già compilati: i target vanno rivisti sul contesto dell'organizzazione, perché un target ereditato non è un obiettivo.`,
      },
      {
        tipo: "avviso",
        tono: "errore",
        titolo: "Il verso sbagliato inverte ogni lettura",
        testo:
          "Per un indicatore che deve calare — reclami, infortuni, consumi — la soglia di attenzione sta **sopra** il target; per uno che deve salire sta sotto. Un verso sbagliato non produce un errore: produce stati, scostamenti e tendenze rovesciati, e un cruscotto che dice «a target» proprio mentre le cose peggiorano.",
      },
      {
        tipo: "avviso",
        tono: "attenzione",
        titolo: "Un cruscotto di soli indicatori reattivi arriva tardi",
        testo:
          "I reattivi registrano ciò che è già accaduto — infortuni, reclami, non conformità — e sono i più facili da compilare perché il dato esiste già. I proattivi misurano la prevenzione: sopralluoghi, mancati infortuni segnalati, formazione in regola. Almeno uno per ambito, altrimenti il sistema misura solo i propri fallimenti.",
      },
      {
        tipo: "avviso",
        tono: "nota",
        titolo: "Un indicatore senza target non è «a target»",
        testo:
          "Resta non rilevato. Nel prototipo un target vuoto veniva letto come zero, e l'indicatore risultava conforme senza che nessuno avesse fissato niente — il ramo della soglia non veniva mai raggiunto. È lo stesso difetto dell'indicatore energetico senza denominatore: un vuoto letto come uno zero è la lettura più lusinghiera possibile di un dato che non c'è.",
      },
      {
        tipo: "prosa",
        testo:
          "Cambiare il criterio di calcolo di un indicatore va annotato: la serie storica o si ricostruisce o si interrompe esplicitamente. Un grafico che prosegue attraverso un cambio di formula confronta due cose diverse e non lo dice.",
      },
    ],
  },

  {
    id: "registri-qas",
    titolo: "I registri",
    minuti: 6,
    sommario: `I ${NUMERI.registriQas} registri che alimentano cruscotto, riesame e stampe.`,
    blocchi: [
      {
        tipo: "prosa",
        testo: `Sono **${NUMERI.registriQas}**: mappa dei processi, parti interessate, consultazione dei lavoratori, aspetti ambientali, pericoli e rischi, obblighi di conformità, autorizzazioni e verifiche periodiche, obiettivi e programmi, gestione delle modifiche, formazione, albo fornitori, prove di emergenza, reclami e soddisfazione, audit interni, non conformità e azioni correttive, incidenti e infortuni.`,
      },
      {
        tipo: "avviso",
        tono: "attenzione",
        titolo: "Date e stati fanno i calcoli",
        testo:
          "Un audit risulta eseguito solo con la data; una non conformità è chiusa nei termini solo con termine e chiusura; un infortunio entra negli indici solo se è classificato come tale e ha i giorni di assenza. Registrazioni incomplete restano nell'elenco e non muovono né cruscotto né riesame — ed è giusto, perché nemmeno in audit muoverebbero qualcosa.",
      },
      {
        tipo: "prosa",
        testo:
          "Il registro delle autorizzazioni porta la scadenza e l'anticipo con cui va avviato il rinnovo: è l'unico posto del sistema dove una data che passa produce un problema che non dipende da noi, e per questo l'anticipo si compila insieme alla scadenza.",
      },
    ],
  },

  {
    id: "motori-qas",
    titolo: "I due motori di valutazione",
    minuti: 7,
    sommario: "Significatività degli aspetti ambientali, e livello di rischio dei pericoli.",
    blocchi: [
      {
        tipo: "formula",
        testo: "significatività = gravità × frequenza × sensibilità del contesto",
      },
      {
        tipo: "avviso",
        tono: "errore",
        titolo: "I criteri assoluti rendono significativo un aspetto a prescindere dal punteggio",
        testo:
          "Una prescrizione legale non pienamente presidiata, un esposto o un reclamo ricevuto, un superamento di limiti, una condizione di emergenza grave: ciascuno rende l'aspetto significativo da solo. Valutare solo il prodotto dei tre fattori porta a classificare non significativa una prescrizione che non si sta rispettando, che è esattamente il caso per cui questi criteri esistono.",
      },
      {
        tipo: "prosa",
        testo:
          "Per ogni aspetto si dichiarano anche la fase del ciclo di vita e la condizione: normale, anomala, di emergenza. Un aspetto valutato solo in condizioni normali descrive l'organizzazione nei giorni in cui tutto funziona.",
      },
      {
        tipo: "formula",
        testo: "livello di rischio = probabilità × gravità",
      },
      {
        tipo: "avviso",
        tono: "attenzione",
        titolo: "La gerarchia dei controlli si dichiara, e fermarsi ai dispositivi va motivato",
        testo:
          "Eliminazione, sostituzione, controlli tecnici, controlli amministrativi, dispositivi di protezione individuale: in quest'ordine. Se non si applicano i livelli superiori serve la ragione, ed è il rilievo più tipico di un audit sulla sicurezza — una valutazione che risolve tutto con i dispositivi non ha guardato il problema, l'ha spostato sulla persona.",
      },
      {
        tipo: "prosa",
        testo:
          "Il rischio residuo si dichiara **dopo** le misure da attuare, con responsabile e termine: prima di allora è una previsione, non una valutazione.",
      },
    ],
  },

  {
    id: "conformita-qas",
    titolo: "La mappa di conformità",
    minuti: 6,
    sommario: `I ${NUMERI.requisitiQas} controlli sui ${NUMERI.capiQas} capitoli, filtrabili per norma.`,
    blocchi: [
      {
        tipo: "prosa",
        testo: `Ogni domanda porta il punto della norma e la marcatura delle **${NUMERI.normeQas} norme** che lo richiedono, la procedura che risponde, lo stato e l'evidenza. Il filtro per norma mostra solo le domande di quel sistema e la sua conformità: è così che chi applica una norma sola vede il proprio perimetro invece di quello di tutti.`,
      },
      {
        tipo: "avviso",
        tono: "errore",
        titolo: "Un requisito applicabile e non valutato pesa ZERO",
        testo:
          "Mediare sui soli valutati farebbe salire la conformità man mano che si saltano i requisiti difficili. Su un documento che precede un audit di certificazione è la differenza fra sapere a che punto si è e credere di esserci.",
      },
      {
        tipo: "avviso",
        tono: "attenzione",
        titolo: "«Non applicabile» con parsimonia",
        testo:
          "È corretto per le norme non applicate e per le esclusioni motivate dichiarate in anagrafica. Usato per evitare una non conformità svuota la mappa: la percentuale sale e non regge alla prima domanda.",
      },
      {
        tipo: "prosa",
        testo:
          "Le domande non conformi e parzialmente conformi sono l'elenco delle azioni prima della certificazione, e l'evidenza annotata è il riferimento che si esibirà. Molte trovano risposta nei registri: annotare il registro è più solido che rimandare a una procedura che descrive l'intenzione.",
      },
    ],
  },

  {
    id: "documenti-qas",
    titolo: "Procedure e documenti prodotti",
    minuti: 5,
    sommario: `Le ${NUMERI.procedureQas} procedure, i ${NUMERI.moduliQas} moduli, e i tre documenti che escono.`,
    blocchi: [
      {
        tipo: "prosa",
        testo: `Il corpus base porta **${NUMERI.procedureQas} procedure** e **${NUMERI.moduliQas} moduli**, dal contesto al miglioramento. Ogni procedura riporta i punti delle norme che copre, e alcune sono specifiche di un solo ambito: chi non applica quella norma le dichiara non applicabili invece di personalizzarle.`,
      },
      {
        tipo: "prosa",
        testo:
          "Il percorso produce tre documenti: il **Riesame di direzione**, che precompila gli elementi in ingresso delle norme con i dati del sistema lasciando alla direzione la colonna delle valutazioni; l'**Analisi ambientale**, con aspetti, impatti, criteri e significatività; e la **Valutazione dei rischi**, con pericoli, gerarchia dei controlli e residuo. Gli ultimi due sono i documenti che un auditor chiede per primi.",
      },
      {
        tipo: "avviso",
        tono: "errore",
        titolo: "Un riesame che elenca i dati non è un riesame",
        testo:
          "Il requisito chiede valutazioni e decisioni: risorse, obiettivi, modifiche. Un documento che riporta gli indicatori senza dire che cosa la direzione ne pensa e che cosa ha deciso non soddisfa il punto della norma, e lo dice il documento stesso.",
      },
    ],
  },

  {
    id: "errori-qas",
    titolo: "La sequenza di impianto, e gli errori più frequenti",
    minuti: 4,
    sommario: "In che ordine si costruisce, e che cosa fa mentire il cruscotto.",
    blocchi: [
      {
        tipo: "elenco",
        voci: [
          "Anagrafica con ruoli, norme applicate, campo di applicazione e ore lavorate.",
          "Registri di contesto: mappa dei processi, parti interessate, obblighi di conformità, autorizzazioni con le scadenze.",
          "I due motori: aspetti ambientali e pericoli valutati per intero; obiettivi collegati agli aspetti significativi e ai rischi più alti.",
          "Indicatori dal set di partenza, target rivisti, prime rilevazioni; procedure approvate in ordine di capitolo.",
          "Mappa valutata su tutte le domande delle norme applicate; audit interno; riesame di direzione.",
        ],
      },
      {
        tipo: "tabella",
        intestazioni: ["Errore", "Conseguenza", "Come si evita"],
        righe: [
          ["Ore lavorate non inserite", "Indici infortunistici vuoti dove servono", "Aggiornarle a ogni periodo"],
          [
            "Verso di miglioramento sbagliato",
            "Stati, scostamenti e tendenze invertiti, in silenzio",
            "Decrescente per ciò che deve calare",
          ],
          ["Solo indicatori reattivi", "Il cruscotto registra gli esiti quando è tardi", "Almeno un proattivo per ambito"],
          [
            "Criterio cambiato senza nota",
            "Serie storica che confronta cose diverse",
            "Annotare la modifica, o interrompere la serie",
          ],
          [
            "Aspetti valutati senza i criteri assoluti",
            "Prescrizioni non presidiate classificate non significative",
            "Compilare i criteri assoluti su ogni aspetto",
          ],
          [
            "Gerarchia dei controlli ferma ai dispositivi senza motivazione",
            "Il rilievo più tipico di un audit sulla sicurezza",
            "Motivare quando i livelli superiori non si applicano",
          ],
          ["Riesame senza valutazioni della direzione", "Requisito non soddisfatto", "Colonna di valutazione e decisioni compilate"],
        ],
      },
    ],
  },
];

File: audio-formazione/distilla-slide/sgiqas.py
# Slide del sistema di gestione integrato qualità, ambiente, sicurezza.
from _scrivi import P, scrivi

SLIDE = {
    "anagrafica-qas": [
        (0, "apertura", dict(
            titolo="Anagrafica", sottotitolo="Due decisioni che pesano su tutto",
            testo="Quanti requisiti dovrai gestire, e se gli indici del cruscotto avranno un numero o resteranno vuoti.",
            agenda=["Le norme applicate", "I ruoli", "Esclusioni ammesse e no", "Le ore lavorate", "L'ordine",
                    "Il campo di applicazione"])),
        (1, "definizione", dict(
            eyebrow="Un elenco, non una scelta unica", titolo="Le norme applicate",
            definizione="Una, due o tutte e {normeQas}: dalla combinazione dipende quali dei {requisitiQas} requisiti entrano nella mappa.",
            punti=[P("Certificato solo sulla qualità", "non si vede contare addosso ambiente e sicurezza")])),
        (2, "split", dict(
            titolo="I ruoli, con nomi veri",
            lead="Sono le persone a cui il sistema attribuirà le azioni: un ruolo vuoto è un'azione senza destinatario.",
            punti=[P("Direzione", "dell'organizzazione"), P("Responsabile", "del sistema"),
                   P("Responsabile", "della prevenzione"), P("Medico", "competente")])),
        (3, "confronto", dict(
            titolo="Le esclusioni",
            a={"h": "Ammessa", "sub": "Un requisito di qualità che non si applica",
               "punti": ["La progettazione, per chi produce su specifica del cliente"]},
            b={"h": "Mai ammessa", "sub": "Ambiente e sicurezza",
               "punti": ["Attività con impatti ambientali significativi", "Luoghi di lavoro"]})),
        (4, "evidenza", dict(
            eyebrow="Il campo che vale più di tutti",
            titolo="Senza ore lavorate, gli indici di sicurezza restano vuoti.",
            testo="Frequenza e gravità non escono sbagliati: non escono. Vuoti nel cruscotto e nel riesame, dove qualcuno andrà a cercarli.")),
        (5, "definizione", dict(
            eyebrow="Un errore silenzioso", titolo="Ore lavorate aggiornate a ogni periodo",
            definizione="Compilate a gennaio e dimenticate, a dicembre gli indici si calcolano su un monte ore vecchio di un anno.")),
        (6, "flusso", dict(
            titolo="La sequenza che funziona",
            passi=[P("Le norme", "determinano il perimetro"),
                   P("Campo ed esclusioni", "da concordare con la direzione"),
                   P("Ruoli e ore lavorate", "raccolta, delegabile a chi ha i dati")])),
        (7, "chiusura", dict(
            titolo="Da portare via",
            punti=[P("Il campo di applicazione", "coincide con quello dei certificati"),
                   P("Le esclusioni", "mai su ambiente e sicurezza"), P("Le ore lavorate", "a ogni periodo")])),
    ],
    "indicatori-qas": [
        (0, "apertura", dict(
            titolo="Gli indicatori", sottotitolo="Quelli che i clienti guardano di più",
            testo="E che si costruiscono peggio: vale la pena capire perché prima di compilarne uno.",
            agenda=["Che cosa porta un indicatore", "Il verso del miglioramento", "I target vuoti",
                    "Reattivi e proattivi", "Cambiare la formula", "La frequenza"])),
        (1, "split", dict(
            titolo="Che cosa porta un indicatore",
            lead="Il set di partenza ne propone {indicatoriQas}, già con formula e riferimenti. I target no: un target ereditato non è un obiettivo.",
            punti=[P("Definizione", "e formula"), P("Fonte", "e frequenza di rilevazione"), P("Valore iniziale", "il punto di partenza"),
                   P("Target", "con il suo verso"), P("Soglia", "di attenzione")])),
        (2, "confronto", dict(
            titolo="Il verso del miglioramento",
            a={"h": "Deve calare", "sub": "Reclami, infortuni, consumi", "punti": ["Soglia di attenzione sopra il target"]},
            b={"h": "Deve salire", "sub": "Soddisfazione, formazione", "punti": ["Soglia di attenzione sotto il target"]})),
        (3, "evidenza", dict(
            eyebrow="Un altro modo in cui un cruscotto mente",
            titolo="Un indicatore senza target non è a target: è non rilevato.",
            testo="Nel prototipo un target vuoto valeva zero, e l'indicatore risultava conforme. Su ogni casella vuota: che si veda che manca.")),
        (4, "cards", dict(
            titolo="Reattivi e proattivi",
            cards=[P("Reattivi", "Infortuni, reclami, non conformità: ciò che è già accaduto."),
                   P("Proattivi", "Sopralluoghi, mancati infortuni, formazione: almeno uno per ambito.")])),
        (5, "definizione", dict(
            eyebrow="Con il cliente", titolo="Rendicontare o governare",
            definizione="I reattivi servono a rendicontare, i proattivi a governare: con i secondi si decide qualcosa lunedì mattina.")),
        (6, "evidenza", dict(
            eyebrow="Cambiare il criterio di calcolo",
            titolo="Si annota, e la serie si ricostruisce o si interrompe.",
            testo="Un grafico che prosegue attraverso un cambio di formula confronta due cose diverse, e non lo dice.")),
        (7, "chiusura", dict(
            titolo="La frequenza di rilevazione",
            punti=[P("Mensile compilato quando capita", "una serie con i buchi"),
                   P("Una serie con i buchi", "non mostra una tendenza"), P("Meglio trimestrale", "e rispettato")])),
    ],
    "registri-qas": [
        (0, "apertura", dict(
            titolo="I registri", sottotitolo="{registriQas} fonti di calcolo, non archivi",
            testo="Cruscotto, riesame e stampe prendono i numeri da qui: un registro incompleto è un calcolo che non si fa.",
            agenda=["Quattro gruppi", "Date e stati", "Da dove cominciare", "Le autorizzazioni",
                    "Che cosa chiede chi controlla", "La gestione delle modifiche"])),
        (1, "cards", dict(
            titolo="Quattro gruppi",
            cards=[P("Contesto e persone", "Processi, parti interessate, consultazione dei lavoratori."),
                   P("Ambiente e sicurezza", "Aspetti, pericoli, obblighi, autorizzazioni."),
                   P("Gestione", "Obiettivi, modifiche, formazione, fornitori, emergenze."),
                   P("Controllo", "Reclami, audit, non conformità, infortuni.")])),
        (2, "tabella", dict(
            titolo="Sono le date e gli stati a fare i calcoli",
            cols=["Registrazione", "Conta solo se"],
            righe=[["Audit eseguito", "Ha la data"],
                   ["Non conformità chiusa nei termini", "Ha termine e data di chiusura"],
                   ["Infortunio negli indici", "È classificato infortunio e ha i giorni di assenza"]])),
        (3, "evidenza", dict(
            eyebrow="Sembra una durezza del prodotto",
            titolo="È quello che succederebbe in audit.",
            testo="Un audit senza data non viene contato come eseguito: il sistema te lo dice prima del verificatore.")),
        (4, "split", dict(
            titolo="Non aprire tutti i registri insieme",
            lead="Comincia da quelli che alimentano gli indicatori scelti; gli altri quando succede la cosa.",
            punti=[P("Un indicatore sui reclami", "apri reclami e soddisfazione"),
                   P("Uno sulla formazione", "apri formazione"),
                   P("Tutti aperti a gennaio", "tutti a metà, e non servono")])),
        (5, "numeri", dict(
            titolo="Le autorizzazioni: l'anticipo va generoso",
            numeri=[{"n": "60 giorni", "h": "Su un rinnovo che ne chiede trenta", "d": "Non costano niente."},
                    {"n": "30 giorni", "h": "Su un rinnovo che ne chiede novanta", "d": "Costano una sospensione dell'attività."}])),
        (6, "cards", dict(
            titolo="Tre richieste di chi controlla",
            cards=[P("L'ultimo audit interno", "E il suo rapporto."),
                   P("Una non conformità dell'anno scorso", "E come è stata chiusa."),
                   P("Le autorizzazioni", "In scadenza nei prossimi sei mesi.")])),
        (7, "definizione", dict(
            eyebrow="Il registro più trascurato", titolo="La gestione delle modifiche",
            definizione="Nessun evento lo attiva da fuori: un macchinario cambiato lo scrive solo chi ha preso l'abitudine di farlo.")),
    ],
    "motori-qas": [
        (0, "apertura", dict(
            titolo="I due motori di valutazione", sottotitolo="Ambiente e sicurezza, due domande diverse",
            testo="Il cuore tecnico del sistema, e funzionano in modo diverso.",
            agenda=["La significatività", "I criteri assoluti", "Le condizioni operative", "Il livello di rischio",
                    "La gerarchia dei controlli", "Il rischio residuo"])),
        (1, "definizione", dict(
            eyebrow="Motore ambientale", titolo="La significatività",
            definizione="Gravità × frequenza × sensibilità del contesto: un prodotto, quindi un fattore basso abbassa tutto.")),
        (2, "punti", dict(
            titolo="I criteri assoluti",
            intro="Ciascuno rende l'aspetto significativo da solo, qualunque cosa dica il punteggio.",
            numerati=False,
            punti=[P("Prescrizione legale", "non pienamente presidiata"), P("Esposto o reclamo", "ricevuto"),
                   P("Limiti", "superati"), P("Emergenza", "in condizione grave")])),
        (3, "evidenza", dict(
            eyebrow="Normale, anomala, emergenza",
            titolo="I problemi ambientali non capitano nei giorni in cui tutto funziona.",
            testo="Lo scarico nei limiti a regime può uscirne durante un avviamento: se l'avviamento non è valutato, non compare da nessuna parte.")),
        (4, "definizione", dict(
            eyebrow="Motore della sicurezza", titolo="Il livello di rischio",
            definizione="Probabilità × gravità. La formula è semplice: il punto interessante viene dopo.")),
        (5, "flusso", dict(
            titolo="La gerarchia dei controlli",
            passi=[P("Eliminazione", "del pericolo"), P("Sostituzione", "con qualcosa di meno pericoloso"),
                   P("Controlli tecnici", "sull'impianto"), P("Controlli amministrativi", "procedure e organizzazione"),
                   P("Protezione individuale", "l'ultima difesa")])),
        (6, "confronto", dict(
            titolo="Il casco e il carico sospeso",
            a={"h": "Il casco", "sub": "Protegge chi lo indossa", "punti": ["Il problema spostato sulla persona"]},
            b={"h": "Togliere il carico sospeso", "sub": "Protegge chiunque passi", "punti": ["Anche chi ha dimenticato il casco"]})),
        (7, "evidenza", dict(
            eyebrow="Il rischio residuo",
            titolo="Si dichiara dopo le misure, non prima.",
            testo="Compilato prima è una previsione: descrive un'azienda che non esiste ancora.")),
        (8, "chiusura", dict(
            titolo="La rivalutazione",
            punti=[P("Una valutazione", "è vera quando è stata fatta"),
                   P("Una linea nuova, un reparto spostato", "la invecchiano"),
                   P("Senza un'occasione fissata", "resta agli atti come se fosse attuale")])),
    ],
    "conformita-qas": [
        (0, "apertura", dict(
            titolo="La mappa di conformità", sottotitolo="{requisitiQas} domande su {capiQas} capitoli",
            testo="Ogni domanda con il punto della norma, la marcatura delle norme che lo richiedono, la procedura, lo stato e l'evidenza.",
            agenda=["La marcatura", "Il filtro per norma", "Non valutato pesa zero", "Il non applicabile",
                    "Le evidenze", "Da dove cominciare"])),
        (1, "confronto", dict(
            titolo="La marcatura",
            a={"h": "Punti comuni", "sub": "Richiesti da tutte e tre",
               "punti": ["Contesto", "Parti interessate", "Competenza", "Audit interno"]},
            b={"h": "Punti propri", "sub": "Di una norma sola", "punti": ["Si vedono solo nel suo filtro"]})),
        (2, "evidenza", dict(
            eyebrow="Il filtro per norma",
            titolo="Ognuno vede il proprio perimetro.",
            testo="Chi applica solo la qualità non si trova la conformità abbassata da domande ambientali che non lo riguardano.")),
        (3, "definizione", dict(
            eyebrow="La regola comune", titolo="Non valutato pesa zero",
            definizione="Mediare sui soli valutati farebbe salire la conformità saltando i requisiti difficili.")),
        (4, "split", dict(
            titolo="Non applicabile, con parsimonia",
            lead="Corretto per le norme che l'azienda non applica e per le esclusioni dichiarate in anagrafica.",
            punti=[P("Per evitare una non conformità", "svuota la mappa"), P("La prima domanda dell'auditor", "è sempre perché")])),
        (6, "cards", dict(
            titolo="Quale evidenza",
            cards=[P("Una procedura", "Descrive l'intenzione: si legge in un minuto e non prova niente."),
                   P("Un registro", "Dimostra l'esecuzione: si apre e mostra le righe.")])),
        (7, "numeri", dict(
            titolo="Una norma alla volta",
            numeri=[{"n": "Tre al 70%", "h": "In ordine di capitolo", "d": "Il giorno della visita."},
                    {"n": "Una al 100%", "h": "Partendo dal suo filtro", "d": "La prima norma da certificare."}])),
    ],
    "documenti-qas": [
        (0, "apertura", dict(
            titolo="I documenti", sottotitolo="{procedureQas} procedure e {moduliQas} moduli",
            testo="Ogni procedura riporta i punti delle norme che copre: si sa sempre che cosa resta scoperto.",
            agenda=["Procedure di un solo ambito", "Tre documenti", "Il riesame", "La colonna delle valutazioni",
                    "Compilarlo in riunione"])),
        (1, "definizione", dict(
            eyebrow="Procedure di un solo ambito", titolo="Non applicabili, non personalizzate",
            definizione="Chi non applica quella norma le dichiara non applicabili: una procedura per un sistema che non esiste confonde chi la troverà.")),
        (3, "cards", dict(
            titolo="Tre documenti",
            cards=[P("Riesame di direzione", "Ingressi precompilati con i dati veri, valutazioni alla direzione."),
                   P("Analisi ambientale", "Aspetti, impatti, criteri, significatività."),
                   P("Valutazione dei rischi", "Pericoli, gerarchia dei controlli, rischio residuo.")])),
        (4, "evidenza", dict(
            eyebrow="Chiesti per primi da un auditor",
            titolo="Analisi ambientale e valutazione dei rischi.",
            testo="Prima ancora del manuale: da lì si capisce se il sistema è stato costruito su quell'azienda, o comprato.")),
        (6, "confronto", dict(
            titolo="Un riesame che elenca i dati non è un riesame",
            a={"h": "Una raccolta di indicatori", "sub": "Non soddisfa il punto", "punti": ["Un rilievo ogni volta"]},
            b={"h": "Valutazioni e decisioni", "sub": "Quello che la norma chiede",
               "punti": ["Le risorse", "Gli obiettivi confermati o cambiati", "Le modifiche al sistema"]})),
        (7, "definizione", dict(
            eyebrow="Una scelta del prodotto", titolo="La colonna delle valutazioni resta vuota",
            definizione="Invece di una frase di comodo: è una domanda posta alla direzione, e se resta vuota lo dice il documento.")),
        (8, "chiusura", dict(
            titolo="Compilare il riesame in riunione",
            punti=[P("Gli ingressi", "già compilati"), P("Le valutazioni", "scritte mentre si discute"),
                   P("Compilato insieme", "è una decisione, non un documento firmato")])),
    ],
    "errori-qas": [
        (0, "apertura", dict(
            titolo="L'ordine, e gli errori", sottotitolo="Sette errori, nessun avviso",
            testo="L'ordine conta perché i due motori dipendono da quello che c'è prima.",
            agenda=["L'ordine", "Nessun avviso", "Quattro errori sugli indicatori", "Motori e riesame",
                    "Il valore del sistema integrato"])),
        (1, "flusso", dict(
            titolo="L'ordine in cui si costruisce",
            passi=[P("Anagrafica", "ruoli, norme, campo, ore lavorate"),
                   P("Registri di contesto", "processi, parti, obblighi, autorizzazioni"),
                   P("I due motori", "con gli obiettivi collegati"),
                   P("Indicatori e procedure", "target rivisti, prime rilevazioni"),
                   P("Mappa, audit, riesame", "sulle norme applicate")])),
        (2, "evidenza", dict(
            eyebrow="Che cosa li accomuna",
            titolo="Nessuno dei sette produce un avviso.",
            testo="Il cruscotto continua a calcolare, e mente.")),
        (3, "tabella", dict(
            titolo="Quattro errori sugli indicatori",
            cols=["Errore", "Effetto"],
            righe=[["Ore lavorate non inserite", "Indici infortunistici vuoti"],
                   ["Verso di miglioramento sbagliato", "Verde mentre si peggiora"],
                   ["Solo indicatori reattivi", "Si registra quando è tardi"],
                   ["Formula cambiata senza annotarla", "Una serie che confronta cose diverse"]])),
        (7, "cards", dict(
            titolo="Gli ultimi tre",
            cards=[P("Aspetti senza criteri assoluti", "Prescrizioni violate classificate non significative."),
                   P("Gerarchia ferma ai dispositivi", "Senza motivazione: il rilievo tipico sulla sicurezza."),
                   P("Riesame senza valutazioni", "Non soddisfa il punto della norma.")])),
        (8, "chiusura", dict(
            titolo="Tre sistemi tenuti insieme",
            punti=[P("Tre manuali, tre audit, tre riesami", "costano il triplo del necessario"),
                   P("I punti comuni", "sono la maggioranza, e si compilano una volta"),
                   P("Mostrarlo al primo incontro", "vende il percorso")])),
    ],
}

scrivi("sgiqas", SLIDE)
